utils/tf_solver.py

```diff
@@ -12,31 +14,42 @@
 class TFSolver(BaseSolver):
     """TF base solver"""
 
-    stopping_strategy = 'callback'
+    stopping_criterion = SufficientProgressCriterion(
+        patience=20, strategy='callback'
+    )
+
+    parameters = {
+        'batch_size': [64],
+    }
 
     # XXX: this should be removed once
     # https://github.com/benchopt/benchmark_resnet_classif/pull/6
     # and
     # https://github.com/benchopt/benchopt/pull/323
     # are merged
-    def skip(self, pl_module, trainer, tf_model, tf_dataset):
+    def skip(self, pl_module, torch_dataset, tf_model, tf_dataset):
         if tf_model is None or tf_dataset is None:
             return True, 'Dataset not fit for TF use'
         return False, None
 
-    def set_objective(self, pl_module, trainer, tf_model, tf_dataset):
+    def set_objective(self, pl_module, torch_dataset, tf_model, tf_dataset):
         self.tf_model = tf_model
         self.tf_model.compile(
             optimizer=self.optimizer,
-            loss='categorical_crossentropy',
+            loss='sparse_categorical_crossentropy',
```

Comment:
Do we know which one is fastest?
I am guessing sparse, but it might be interesting to have in a comment a link specifying it.